<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Py_Yolo V1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../다운로드.png</normaloff>../../../../다운로드.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: pink;
QAbstractScrollArea.background:white;</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BT_Character_Name_Search">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #dc3545;
border-color: #dc3545;
background-color: whitesmoke;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_Charater_Name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid white; 
color: #5B2386;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Character_Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Character_Name">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>510</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid white; 
color: #dc3545;
border-radius: 10px;

background-color: whitesmoke;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Input Character_Name</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="PB_Character">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>550</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 150);
 border: 2px solid white; 
color: white;
border-radius: 5px;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_Exit">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #dc3545;
border-color: #dc3545;
background-color: whitesmoke;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_Image_Save">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #dc3545;
border-color: #dc3545;
background-color: whitesmoke;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Image Save</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Log">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>50</y>
      <width>291</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 170);
border: 2px solid white;
color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
